### PR TITLE
fix: 趋势分析表 conditions 增加容错能力

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-react/playground/config.ts [hunk 1]
```diff
@@ -117,25 +117,27 @@ export const mockGridAnalysisOptions: S2Options = {
...
-                fill: isUpDataValue(value) ? '#FF4D4F' : '#29A294',
-              };
-            },
-          },
-        },
       },
     },
   },
+  conditions: {
+    text: [
```
Comment (wjgogogo) [inline packages/s2-react/playground/config.ts:124]: 网格分析表 conditions也需要挪到相应位置，否则会报错：
image

Comment (xingwanying) [reply packages/s2-react/playground/config.ts:124]: 我去搞忘记了……感谢大佬 🙏

**Review by xingwanying — DISMISSED** [review]
LG

**Review by stone-lyl — DISMISSED** [review]

**Review by lijinke666 — DISMISSED** [review]

**Review by lijinke666 — DISMISSED** [review]

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**